internal/handlers/pg/pgdb/trace.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgdb
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+)
+
+var tracer = otel.Tracer("internal/handlers/pg/pgdb")
+
+// debugTracer implements pgx.QueryTracer. It is used to add traces to Query,
+// QueryRow, and Exec calls in debug builds.
+type debugTracer struct{}
+
+// TraceQueryStart adds a span to Query, QueryRow, and Exec calls.
+func (t *debugTracer) TraceQueryStart(ctx context.Context, _ *pgx.Conn, _ pgx.TraceQueryStartData) context.Context {
+	ctx, _ = tracer.Start(ctx, "TraceQueryStart")
+	return ctx
+}
+
+// TraceQueryEnd ends the span started by TraceQueryStart.
+func (t *debugTracer) TraceQueryEnd(ctx context.Context, _ *pgx.Conn, _ pgx.TraceQueryEndData) {
+	span := trace.SpanFromContext(ctx)
```

Comment:
Same here: let's use some data from `TraceQueryEndData` as span attributes to have more info available on the Jaeger UI.

Reply:
Done. Thank you!